README.adoc

```diff
@@ -197,6 +197,57 @@ go test ./...
 custom configuration CLI supports `+--config+` flag. Sample
 configuration can be found in [config.toml.SAMPLE](configs/config.toml.SAMPLE).
 
+==== Storage
+Users configure the root storage directory at `Storage.DataDir`. The storage layout looks like:
+```
+.
+├── archive
+│   ├── 0x08E0__ADDRESS__7695fCeb63A70FF85454B683E
+│   │   └── membership_046__MEMBERSHIP_ADDRESS__bfa1e3535ae57de
+│   ├── 0x33ed__ADDRESS__Bb76baD50EeedfE422793E4ED
+│   │   └── membership_046__MEMBERSHIP_ADDRESS__bfa1e3535ae57de
+├── bitcoin
+│   ├── derivation_indexes
+│   │   ├── xpub_zCAk9sk1
+│   │   │   ├── 172
+│   │   │   ├── 3544
+│   │   │   ├── 39
+│   │   │   ├── 5
+│   │   │   ├── 5090
+│   │   │   └── 513
+│   │   ├── ypub_6Z2Yy4ZH
+│   │   │   ├── 1692
+│   │   │   ├── 223
+│   │   │   ├── 48
+│   │   │   └── 8982
+│   │   └── zpub_VDaZuQT9
+│   │       ├── 112
+│   │       ├── 6311
+│   │       ├── 6999
+│   │       └── 8559
```

Comment:
Isn't it just one file per address stored with the latest used index?

![image](https://user-images.githubusercontent.com/10741774/125082333-a0b7c580-e0b6-11eb-8534-855e9c0eda5d.png)

Reply:
Oh, yup! I forgot that we changed it to that and grabbed the folder structure from the original PR